fe/fe-core/src/test/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2Test.java

```diff
@@ -491,4 +509,139 @@ public void testDatabaseStmt() throws Exception {
                 "revoke ALTER on database " + testDbName + " from test",
                 "Access denied for user 'test' to database '" + testDbName + "'");
     }
+    
+    @Test
+    public void testShowNodeStmt() throws Exception {
+
+        verifyGrantRevoke(
```

Comment:
these three show statements also need to test OPERATE privilege.

Reply:
done